packages/kotti-ui/source/kotti-navbar/KtNavbar.vue

```diff
@@ -111,10 +178,7 @@ $narrow-navbar-width: 3.4rem;
 	--navbar-color-light: var(--primary-20);
 	--navbar-color-active: var(--white);
 
-	--user-menu-border: var(--primary-80);
-	--user-menu-background-active: var(--primary-70);
 	--user-menu-background: var(--primary-60);
-	--user-menu-color: var(--primary-10);
 
 	position: relative;
```

Comment:
is this needed now? there's no need for controlling absolutes/relatives, tippy handles that, no?

Reply:
I don't think this is needed. But not because of tippy (as this is the Navbar itself, not the dropdown for mobile) but because of `kt-container`.

I can remove, but it is a breaking change for users that do not use `.kt-container`

Reply:
sorry, not `.kt-container` is needed, strictly only a flexbox is needed as a container.

Reply:
I'm slightly confused by this comment :D, I don't know what you mean